lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -959,7 +959,9 @@ namespace Js
         {
             return FALSE;
         }
-        return propertyDescriptor.IsWritable();
+
+        // If property descriptor has getter/setter we should check if writable is specified before checking IsWritable
+        return propertyDescriptor.WritableSpecified() ? propertyDescriptor.IsWritable() : FALSE;
```

Comment:
Yes I was hitting the assert, same assert was there in PDM based debugger as well but now I don't remember the exact test.